builtin/game/chat.lua

```diff
@@ -1,27 +1,41 @@
 -- Minetest: builtin/game/chat.lua
 
+-- Helper function that implements search and replace without pattern matching
+-- Returns the string and a boolean indicating whether or not the string was modified
+local function safe_gsub(s, replace, with)
```

Comment:
Unrelated changes. Please undo.

Reply:
My fork of Minetest wasn't updated, so, I just copy-paste current `chat.lua` into my fork.
This is why Git shows unrelated changes here.

Reply:
Still not fixed.